content/en/docs/setup/getting-started/index.md

```diff
@@ -293,7 +293,14 @@ _Docker For Desktop:_
 $ export INGRESS_HOST=127.0.0.1
 {{< /text >}}
 
-_Other environments (e.g., IBM Cloud Private, etc.):_
+_IBM Cloud Kubernetes Service:_
```

Comment:
Please move this up, before `Docker For Desktop` (right after the GKE instructions).

Reply:
+1 :) thanks @frankbu